pkg/controllers/user/rbac/namespace_handler.go

```diff
@@ -118,6 +119,34 @@ func (n *nsLifecycle) ensurePRTBAddToNamespace(ns *v1.Namespace) (bool, error) {
 			return false, errors.Wrapf(err, "couldn't ensure binding %v in %v", prtb.Name, ns.Name)
 		}
 	}
+
+	if ns.Labels["field.cattle.io/projectId"] == "" {
```

Comment:
This label is not the source of true. A corresponding annotation is. Must use that